app/src/main/res/xml/searchable.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<searchable xmlns:android="http://schemas.android.com/apk/res/android"
+    android:label="@string/app_name"
+    android:hint="@string/lbl_search_hint" 
```

Comment:
I think we want a different string for this one as the current one says "Search text (select for keyboard)". The Android docs  recommend something like "Search Jellyfin" or "Search your media" etc.

https://developer.android.com/guide/topics/search/searchable-config